platform/view/services/storage/driver/sql/query/pagination/save_load.go

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
no need to add an underscore in the name of the file. Go uses the underscore character in specific cases like testing. Better not to abuse it.